metricbeat/values.yaml

```diff
@@ -250,6 +250,12 @@ updateStrategy: RollingUpdate
 nameOverride: ""
 fullnameOverride: ""
 
+# is used only when kube_state_metrics.enabled: false
+kube_state_metrics_host: ""
```

Comment:
What about having `kube_state_metrics.host` instead of `kube_state_metrics_host`?